lib/src/widgets/utils/ds_header.widget.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class DSHeader extends StatelessWidget implements PreferredSizeWidget {
+  final String title;
+  final String? subtitle;
+  final String? customerName;
+  final List<Widget>? actions;
+  final Widget? leading;
+
+  const DSHeader({
+    Key? key,
+    required this.title,
+    this.subtitle,
+    this.customerName,
+    this.actions,
+    this.leading,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      elevation: 1,
+      backgroundColor: DSColors.neutralLightSnow,
+      actions: actions,
+      leadingWidth: 25.0,
+      leading: _buildLeading(context),
```

Comment:
Makes sense to set ```false``` on ```automaticallyImplyLeading``` property?
```suggestion
      leading: _buildLeading(context),
      automaticallyImplyLeading: false,
```

Reply:
We can set `automaticallyImplyLeading` to true to set the back button icon, but its not equals the DS Design system.

Android:
<img width="94" alt="image" src="https://user-images.githubusercontent.com/14128784/188146950-d03f3ea0-b0ff-429e-bb8f-0566fb3057a0.png">

iOS:
<img width="94" alt="image" src="https://user-images.githubusercontent.com/14128784/188147400-f7557aaa-0aac-4e73-b8d8-053c7fb73b02.png">